.github/workflows/check_version.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/sh
 
 version=$1
+path=$2
 
-packageversion=$(cat $2/package.json | grep -Eo '\"version\": \"[[:digit:]]+\.[[:digit:]+\.[[:digit:]]+\"')
-formattedversion="\"version\": \"$version\""
-
-if [ "$formattedversion" = "$packageversion" ]; then
-  exit 0
+if [ "${path: -5}" = .json ]; then
+  # json
+  package_version=$(cat $path | grep -Eo '"version": "[[:digit:]]+\.[[:digit:]+\.[[:digit:]]+"')
+  expected_version="\"version\": \"$version\""
+elif [ "${path: -3}" = .go ]; then
+  # go
+  package_version=$(cat $path | grep -Eo 'Version[[:blank:]]+= "[[:digit:]]+\.[[:digit:]+\.[[:digit:]]+"')
+  expected_version="Version = \"$version\""
 else
+  echo Invalid file $path
+  exit 2
+fi
+
+#get rid of unnecessary spaces
```

Comment:
minor: comment format, and I don't really get why this is necessary from the comment - maybe its late

Reply:
Because in the `version.go` file there are a lot of spaces before the equals sign: `Version             = "0.4.1"`
This line change it to `Version = "0.4.1"`

Reply:
Okey. Thanks!